less/_components/classes-typography.less

```diff
@@ -142,17 +142,40 @@ blockquote{
 // Example: eclipse.org/community/eclipse_newsletter/
 .heading-underline {
   position:relative;
-  font-size:2em;
+  font-size: @heading-underline-font-size;
   text-transform: uppercase;
-  font-weight:200;
-  margin:50px 0 20px;
+  font-weight: @heading-underline-weight;
+  margin: @heading-underline-margin;
   &:after {
-    border-bottom:4px solid #f7941d;
+    border-bottom: @heading-underline-border;
     content:"";
     display:block;
     width:100px;
     margin:10px auto 0;
   }
+  
+  &.extended:after {
+    width: 300px;
+    max-width: 75%;
+    height: @heading-underline-thickness;
+      
+    @media (min-width: 769px) {
+      width: 400px;
+    }
+  }
+  &.left {
```

Comment:
I would expect a developer to use text-left|.text-right|... here.

Reply:
What do you mean? Should I add additional functionality to text-left and text-right in context of being paired with the header underline?